python/dgl/dataloading/async_transferer.py

```diff
@@ -0,0 +1,85 @@
+""" API for transferring data to/from the GPU over second stream.A """
```

Comment:
Could you add some module docstring on code examples of `Transfer` and `AsyncTransferer`? Please also add it to the API doc page [here](https://github.com/dmlc/dgl/blob/master/docs/source/api/python/dgl.dataloading.rst).

Reply:
Added in b1765ce